torch/onnx/verification.py

```diff
@@ -243,6 +247,238 @@ def compare_ort_pytorch_model_with_input(input_args, input_kwargs):
             compare_ort_pytorch_model_with_input(test_input_args, {})
 
 
+class ExportOptions:
+    """Arguments used by :func:`torch.onnx.export`.
+
+    TODO: Adopt this in `torch.onnx.export` api to replace keyword arguments.
+    """
+
+    export_params: bool = True
+    verbose: bool = False
+    training: _C_onnx.TrainingMode = _C_onnx.TrainingMode.EVAL
+    input_names: Optional[Sequence[str]] = None
+    output_names: Optional[Sequence[str]] = None
+    operator_export_type: _C_onnx.OperatorExportTypes = _C_onnx.OperatorExportTypes.ONNX
+    opset_version: Optional[int] = None
+    do_constant_folding: bool = True
+    dynamic_axes: Optional[Mapping[str, Union[Mapping[int, str], Sequence[int]]]] = None
+    keep_initializers_as_inputs: Optional[bool] = None
+    custom_opsets: Optional[Mapping[str, int]] = None
+    export_modules_as_functions: Union[bool, Set[Type[torch.nn.Module]]] = False
+
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+
+class _GraphDiff:
+    """A class to represent the difference between two graphs."""
+
+    def __init__(self, graph_a: _C.Graph, graph_b: _C.Graph):
+        """Construct a _GraphDiff object.
+
+        Args:
+            graph_a (_C.Graph): First graph to compare.
+            graph_b (_C.Graph): Second graph to compare.
+        """
+        self.graph_a = graph_a
+        self.graph_b = graph_b
+
+    def __str__(self):
+        """See function :func:`_graph_diff`."""
+        return self.diff_report()
+
+    def _indent(self, s):
```

Comment:
Also consider param naming:

```suggestion
    def _indent(self, lines: str):
```